app/models/page.rb

```diff
@@ -2,26 +2,8 @@ class Page < ActiveRecord::Base
 
   validates_uniqueness_of :id
   has_many :collected_pages
-  
-  include Elasticsearch::Model
-  include Elasticsearch::Model::Callbacks
-  
-  index_name Rails.application.class.parent_name.underscore
-  document_type self.name.downcase
-  
-  settings index: { number_of_shards: 10} do
-  mapping dynamic: false do
-    indexes :scientitfic_name, type: :varchar 
-    indexes :suggest, {
-      type: 'completion',
-      analyzer: 'lowercase',
-      search_analyzer: 'lowercase',
-      payloads: 'true',
-      }
-  end
-end
 
-  searchkick word_start: [:scientific_name]
+  searchkick word_start: ["name_string"]
```

Comment:
why do you change the "scientific_name" to "name_string"? No need for this given calling separately!